src/Components/Exit.js

```diff
@@ -2,14 +2,15 @@ import React, { useState, useEffect } from 'react'
 import App from './App';
 import '../Styles/Exit.css';
 
-let levelComplete = false;
+export let levelComplete = false;
 
 const Exit = ({ exitReached }) => {
   const [levelOneComplete, setLevelOneComplete] = useState(false);
```

Comment:
I think Redux should help this Exit tile function better by handling state change.